app/models/manageiq/providers/ibm_cloud/inventory/collector/power_virtual_servers/target_collection.rb

```diff
@@ -0,0 +1,125 @@
+class ManageIQ::Providers::IbmCloud::Inventory::Collector::PowerVirtualServers::TargetCollection < ManageIQ::Providers::IbmCloud::Inventory::Collector::PowerVirtualServers
+  def initialize(_manager, _target)
+    super
+
+    parse_targets!
+  end
+
+  def availability_zones
+    []
+  end
+
+  def images
+    @images ||= begin
+      references(:miq_templates).map do |ems_ref|
```

Comment:
@jaywcarman you don't need the `begin` here because there is only a single "line" of code here so
```
@images ||= references(:miq_templates).map do |ems_ref|
   ...
end.compact
```

Reply:
💡now it makes sense!